bsip-0081.md

```diff
@@ -28,23 +28,28 @@ them to attract trading activities.
 
 There is already a flag `market_fee_percent` in asset options.
 
-Add a new flag `taker_fee_percent` into asset options, assign
-`market_fee_percent` to `taker_fee_percentage` at the consensus
-upgrade time for existing assets. The new flag can only be set
+Add a new optional flag `taker_fee_percent` into asset options, 
+which can only be set
 or updated by asset owners after the consensus upgrade.
+The valid range of the new flag is `[0%, 100%]`.
 
 Before the consensus upgrade, when an order buying that asset
 got filled, the amount `bought_amount * market_fee_percent`
-will go to the asset's accumulated fees.
+(capped by `max_market_fee`)
+will enter the asset's market fee sharing process.
 
 After the consensus upgrade, when an order buying the asset
 got filled,
-* if the order is a *maker*, the amount
-  `bought_amount * market_fee_percent` will go to the
-  asset's accumulated fees;
-* otherwise (if the order is a *taker*), the amount
-  `bought_amount * taker_fee_percent` will go to the
-  asset's accumulated fees.
+* if the order is a *maker*, or the `taker_fee_percent` flag
+  is not set in the asset, the amount
+  `bought_amount * market_fee_percent` 
+  (capped by `max_market_fee`)
+  will enter the asset's market fee sharing process;
+* otherwise (if the order is a *taker* and the
```

Comment:
Would it be more precise to include 

"otherwise (the order is a *taker*) the amount `bought_amount * order_fee_percent` (capped by `max_market_fee`) will enter the asset's market fee sharing process where `order_fee_percent` is `taker_fee_percent` if it is set, otherwise `market_fee_percent`"

Reply:
I agree that this is the same idea.  It's a matter of different ways of explaining it.  I support whichever text will be most clear.